lib/utils.pm

```diff
@@ -280,7 +280,6 @@ sub integration_services_check {
         assert_script_run('systemctl list-unit-files | grep hv_fcopy_daemon.service');
     }
     elsif (check_var('VIRSH_VMM_FAMILY', 'vmware')) {
-        integration_services_check_ip;
```

Comment:
Why is this gone?

Reply:
it is not. it was twice for if..else. it was moved at the top

Reply:
Sorry :)